ds000001.jsonld

```diff
@@ -54,970 +50,1056 @@
     "hasAssessment": {
       "@id": "nb:hasAssessment"
     },
+    "Session": "nb:Session",
```

Comment:
This is just here because we include all classes, right?

Reply:
Yes!